src/ray/gcs/gcs_server/gcs_server.cc

```diff
@@ -69,15 +69,17 @@ GcsServer::GcsServer(const ray::gcs::GcsServerConfig &config,
       periodical_runner_(io_context_provider_.GetDefaultIOContext()),
       is_started_(false),
       is_stopped_(false) {
-  // Init GCS table storage.
+  // Init GCS table storage. Note this is on the default io context, not the one with
+  // GcsInternalKVManager, to avoid congestion on the latter.
```

Comment:
what does this mean?

Reply:
Meaning gcs_table_storage_ should always go with GetDefaultIOContext, not with GetIOContext<GcsInternalKVManager>()

Reply:
> to avoid congestion on the latter.

What does this mean?

Reply:
If gcs_table_storage_ operations go with `GcsInternalKVManager`'s, it may slow down GcsInternalKVManager